frame/collective/src/lib.rs

```diff
@@ -185,69 +329,185 @@ decl_module! {
 		/// - `new_members`: The new member list. Be nice to the chain and
 		//	provide it sorted.
 		/// - `prime`: The prime member whose vote sets the default.
+		/// - `old_count`: The number of previous members. (Used for weight estimation.)
```

Comment:
I think we should just ask an upper_bound. that allows to give a margin and if another members is added in between then it is still valid.

Reply:
Is this a naming/phrasing issue? Because the implementation only checks that `old_count` is big enough, not that it matches exactly.

Reply:
ah ok, maybe the documentation should that an upper bound is accepted, name if fine